sdk/src/asset_handlers/jpeg_io.rs

```diff
@@ -80,7 +87,8 @@ fn add_required_segs(asset_path: &std::path::Path) -> Result<()> {
             let mut no_bytes: Vec<u8> = vec![0; 50]; // enough bytes to be valid
             no_bytes.splice(16..20, C2PA_MARKER); // cai UUID signature
             let aio = JpegIO {};
-            aio.save_cai_store(asset_path, &no_bytes)?;
+            aio.write_cai(stream, &no_bytes)?;
+            stream.seek(SeekFrom::Start(0))?;
```

Comment:
Why is this last `.seek` necessary? Shouldn't the subsequent caller be responsible for positioning the streem where it needs it? Or is there an invariant that the stream is always at start-of-file?

Reply:
It just seemed like a good precaution, but it isn't necessary